x-pack/plugins/apm/public/tutorial/config_agent/index.tsx

```diff
@@ -125,7 +129,7 @@ function TutorialConfigAgent({
             onChange={(newSelectedOption) =>
               setSelectedOption(newSelectedOption)
             }
-            fleetLink={fleetLink}
+            fleetLink={data.isFleetEnabled ? fleetLink : undefined}
```

Comment:
Better to keep this as is:

```suggestion
            fleetLink={fleetLink}
```

And the move this logic to `fleetLink`:
```ts
const fleetLinks = getFleetLink()


function getFleetLink(isFleetEnabled: boolean, hasFleetAgents: boolean) {
  // ...
}
```